ui/apps/platform/cypress/integration/configmanagement/ConfigurationManagement.helpers.js

```diff
@@ -153,13 +153,7 @@ function opnameForPrimaryAndSecondaryEntities(entitiesKey1, entitiesKey2) {
 }
 
 const routeMatcherMapForConfigurationManagementDashboard = getRouteMatcherMapForGraphQL([
-    'numPolicies',
-    'numCISControls',
-    'policyViolationsBySeverity',
-    'runStatuses',
     'complianceByControls',
-    'usersWithClusterAdminRoles',
-    'secrets',
```

Comment:
Previously when the scan was completed we would completely empty the GraphQL cache. The updated code use a more fine-grained method for updating stale queries. Due to this, the only query that is refetched on this page is the `'complianceByControls'` query.